sovtokenfees/sovtokenfees/main.py

```diff
@@ -10,6 +13,21 @@ def integrate_plugin_in_node(node):
         ThreePhaseCommitHandler
     from sovtoken import TOKEN_LEDGER_ID
     from sovtoken.client_authnr import TokenAuthNr
+    from indy_common.authorize.auth_constraints import ConstraintsSerializer
+    from common.serializers.serialization import domain_state_serializer
+    from indy_common.authorize.auth_map import auth_map
+    from indy_common.authorize.auth_actions import AuthActionAdd
+    from indy_common.authorize.auth_map import trust_anchor_or_steward_or_trustee_constraint
+
+    set_fees_for_ledger = AuthActionAdd(txn_type=SET_FEES,
+                                        field='*',
+                                        value='*')
+    auth_map.update({set_fees_for_ledger.get_action_id(): trust_anchor_or_steward_or_trustee_constraint})
+
+    serializer = ConstraintsSerializer(domain_state_serializer)
+    node.add_auth_rules_to_config_state(state=node.getState(CONFIG_LEDGER_ID),
```

Comment:
I think, that we shouldn't add all of default authentication rules into config state. Config state is used only for changed default rules. I think, that we should update auth_map into node's write_req_validator. May be we should add a method like "add_default_rule" for WriteRequestValidator class, which would add new auth rule into internal auth_map dictionary.